copy/multiple.go

```diff
@@ -140,8 +162,34 @@ func (c *copier) copyMultipleImages(ctx context.Context, policyContext *signatur
 				UpdateDigest:    updatedManifestDigest,
 				UpdateSize:      int64(len(updatedManifest)),
 				UpdateMediaType: updatedManifestType})
-		default:
-			return nil, fmt.Errorf("copying image: invalid copy operation %d", instance.op)
+		case instanceCopyClone:
+			c.Printf("Replicating image %s (%d/%d)\n", instance.sourceDigest, i+1, len(instanceCopyList))
+			unparsedInstance := image.UnparsedInstance(c.rawSource, &instanceCopyList[i].sourceDigest)
+			instanceDetails, err := updatedList.Instance(instanceCopyList[i].sourceDigest)
+			if err != nil {
+				return nil, fmt.Errorf("Replicating image %d/%d from manifest list: %w", i+1, len(instanceCopyList), err)
+			}
+			addCompressionAlgorithms := []compressionTypes.Algorithm{}
+			optionsClone := options
+			if instance.compressionVariant.Algorithm != nil {
+				// copy options with new compression format
+				optionsClone.DestinationCtx.CompressionFormat = instance.compressionVariant.Algorithm
```

Comment:
https://github.com/containers/image/pull/1987#discussion_r1222035562 is still outstanding.